lens-data/lens/private/syntax/stx.rkt

```diff
@@ -211,7 +223,13 @@
   (define 1* #'1)
   (define 2* #'2)
   (define 3* #'3)
-  (test-case "syntax-e-lens and stx-e-lens"
+  (test-case "restore-stx"
+    (let ([one-dot-two-three (restore-stx #'(a . (b c)) '(1 2 3))])
```

Comment:
Is there a reason this `let` can't be a `define`? The Racket style guide recommends define over let to reduce nesting, and the enclosing test-case form ensures the binding is scoped to this test either way.